tests/js/client/shell/api/coordinator-metrics-cluster.js

```diff
@@ -96,13 +126,64 @@ function CoordinatorMetricsTestSuite() {
 ////////////////////////////////////////////////////////////////////////////////
 /// @brief CoordinatorMetricsTestSuite tests
 ////////////////////////////////////////////////////////////////////////////////
+    testMetricsParameterValidation: function () {
+      let coordinators = getEndpointsByType("coordinator");
+      let dbservers = getEndpointsByType("dbserver");
+      let servers = [coordinators, dbservers];
+      for (let j = 0; j < servers.length; j++) {
+        for (let i = 0; i < servers[j].length; i++) {
+          let server = servers[j][i];
+          let res = getRawMetric(server, '?mode=invalid');
+          assertEqual(res.code, 400);
+        }
+        for (let i = 0; i < servers[j].length; i++) {
+          let server = servers[j][i];
+          let res = getRawMetric(server, '?type=invalid');
+          assertEqual(res.code, 400);
+        }
+        for (let i = 0; i < servers[j].length; i++) {
+          let server = servers[j][i];
+          let res = getRawMetric(server, '?type=invalid&mode=invalid');
+          assertEqual(res.code, 400);
+        }
+      }
+      for (let i = 0; i < coordinators.length; i++) {
+        let server = coordinators[i];
+        let res = getRawMetric(server, '?serverId=invalid');
+        assertEqual(res.code, 404);
+      }
+      for (let i = 0; i < coordinators.length; i++) {
+        let server = coordinators[i];
+        let res = getRawMetric(server, '?serverId=invalid&mode=invalid');
+        assertEqual(res.code, 400);
+      }
+      for (let i = 0; i < coordinators.length; i++) {
+        let server = coordinators[i];
+        let res = getRawMetric(server, '?serverId=invalid&type=invalid');
+        assertEqual(res.code, 400);
+      }
+      for (let i = 0; i < coordinators.length; i++) {
+        let server = coordinators[i];
+        let res = getRawMetric(server, '?serverId=invalid&type=invalid&mode=invalid');
+        assertEqual(res.code, 400);
+      }
+    },
+
+    testMetricsLocalMode: function () {
+      let coordinators = getEndpointsByType("coordinator");
+      for (let i = 0; i < coordinators.length; i++) {
+        let coordinator = coordinators[i];
+        let res = getRawMetric(coordinator, '?mode=local');
+        assertEqual(res.code, 200);
```

Comment:
Yes, that test about parsing mode parameter

Reply:
This test just check parameter validation

Reply:
I cannot assume it's value because it just common prometheus metrics.
For example maybe something like this more desirable
```js
let res1 = getRawMetric(coordinator, '?mode=local');
assertEqual(res1.code, 200);
let res2 = getRawMetric(coordinator, '');
assertEqual(res2.code, 200);
assertEqual(res1.body, res2.body);
```

But it's not true for some queue metrics etc